### PR TITLE
Add a dry-run param to args validation

### PR DESCRIPTION
**Pull request by ramo-j**

### CODE REVIEW

File: dftimewolf/cli/dftimewolf_recipes.py [hunk 1]
```diff
@@ -310,7 +310,7 @@ def ParseArguments(self, arguments: List[str]) -> None:
 
     self._state.command_line_options = vars(self._command_line_options)
 
-  def ValidateArguments(self) -> None:
+  def ValidateArguments(self, dry_run: bool=False) -> None:
```
Comment (sydp) [inline dftimewolf/cli/dftimewolf_recipes.py:313]: Update docstrings?

**Review by sydp — DISMISSED** [review]
Minor nit but lgtm

File: dftimewolf/cli/dftimewolf_recipes.py (hunk 1 above)
Comment (ramo-j) [reply dftimewolf/cli/dftimewolf_recipes.py:313]: Yeah, good call. Updated.

**Review by sydp — APPROVED** [review]
Lgtm

**Pull request merged**